internal/mempool/mempool.go

```diff
@@ -94,15 +94,14 @@ func NewTxMempool(
 	logger log.Logger,
 	cfg *config.MempoolConfig,
 	proxyAppConn proxy.AppConnMempool,
-	height int64,
 	options ...TxMempoolOption,
 ) *TxMempool {
 
 	txmp := &TxMempool{
 		logger:        logger,
 		config:        cfg,
 		proxyAppConn:  proxyAppConn,
-		height:        height,
+		height:        -1,
```

Comment:
Probably missing it, but where is this set now?

Reply:
This is only used in a logging statement and it's updated in `Update()`